<launch>
  <arg name="robot_wrench_topic" default="/mur620b/UR10_r/wrench"/>
  
<node pkg="rosservice" type="rosservice" name="zero_ftsensor" args="call /mur620b/UR10_r/ur_hardware_interface/zero_ftsensor"/>
<node name="wrench_subscriber" pkg="admittance_controller" type="wrench_subscriber.py" output="screen">
    <rosparam param="robot_wrench_topic" subst_value="True">$(arg robot_wrench_topic) </rosparam>
</node>


</launch>